package.json

```diff
@@ -34,7 +34,7 @@
     "registry": "https://registry.npmjs.org"
   },
   "scripts": {
-    "build": "tsup",
+    "build": "cd packages/api-client && pnpm build && cd ../../ && tsup && pnpm --filter '@repo/*' recursive run --sort build",
```

Comment:
🟡 @crherman7 I added this as a temporary solution before merging your work with the migration. but let me know if it's not necessary

Reply:
yeah I will need to test this out

Reply:
yeah this workaround does make sense for the temporary workflow